keyboards/handwired/boss566y/redragon_vara/readme.md

```diff
@@ -0,0 +1,14 @@
+# Handwired Redragon Vara
+
+![Redragon](https://imgur.com/mGJkiZJ.jpg)
```

Comment:
It looks like a split on this picture. Is that on purpose?

Reply:
It was split in the image for my ease in referencing where the break in the rows were when wiring the keyboard

Reply:
Can you upload a picture that represents the layout faithfully?

Reply:
Image has been changed